src/agent/__tests__/MessageSender.test.ts

```diff
@@ -35,7 +36,12 @@ class DummyOutboundTransporter implements OutboundTransporter {
 }
 
 class DummyTransportSession implements TransportSession {
-  public readonly type = 'dummy'
+  public readonly type = 'http'
+  public keys: EnvelopeKeys | undefined
```

Comment:
wouldn't this be possible?

```suggestion
  public keys?: EnvelopeKeys
```

Reply:
Yes, that works 👍